### ISSUE TITLE
Loot error

### ISSUE THREAD

**Issue by Cystallia** [loot/skyrim#353]
TL;DR - Installed Skyrim SE and mods years ago and used Mod Organiser to do so, only had a base knowledge so only basic stuff, fast forward to now, I can no longer remember how to use Mod Organiser and couldn't get Loot to cooperate with Mod Organiser, so I couldn't figure out a load order when I added new mods hence game not even loading. Switch to Vortex since it seemed simpler, all good for a while but occasional crashes. Now, crashes happen about every 20 mins when playing but game log can't tell me which mod is causing an issue, so I try Loot again and it magically works and points out several mistakes within my mod order etc. Change, get patches, etc., then Loot suggests I update which I do, and now I get this error in the debuglog:

[23:48:50.444000] [warning]: Blocking load of resource at https://fonts.googleapis.com/css?family=Roboto+Mono:400,700|Roboto:400,300,300italic,400italic,500,500italic,700,700italic
[23:48:51.947000] [error]: Exception while executing query: Git operation failed. Details: -3; the reference 'refs/heads/v0.14' cannot be peeled - Cannot retrieve reference target: libgit2 error
[23:48:51.948000] [error]: Chromium console message from http://loot/ui/app.bundle.js at line 5761: Error: Oh no, something went wrong! You can check your LOOTDebugLog.txt (you can get to it through the main menu) for more information.
    at onFailure (http://loot/ui/app.bundle.js:5761:9231)

Found one person who had a similar problem, and the solution they got was a windows 7 update regarding TLS 1.2, but I've checked and I have this update so not sure what the issue is. Any ideas?

**Comment by Ortham** [comment]
I'm not sure why the reference can't be peeled, but if you're using the latest version of LOOT, you should be using the `v0.15` masterlist branch, so try changing that in settings.

**Comment by MacSplody** [comment]
@Cystallia Did you resolve this? Closing for now as there's been no update.
